sass.py

```diff
@@ -608,18 +628,21 @@ def _get_file_arg(key):
 
     importers = _validate_importers(kwargs.pop('importers', None))
 
+    indented = kwargs.pop('indented', False)
+    if not isinstance(indented, bool):
+        raise TypeError('indented must be bool, not ' +
+                        repr(source_comments))
```

Comment:
hmmm, won't this now cause `compile(...)` with `filename='foo.sass'` to default to `indented=False`?

Reply:
might be worth a test so we don't accidentally this in the future as well